cmd/snap-serialize.go

```diff
@@ -399,14 +399,16 @@ func (s *snapshotDeserializer) SkipBucketEntries() *probe.Error {
 // BucketEntries will return all bucket entries.
 // The channel will be closed when there are no more entries.
 // If an error occurs it will be returned and the channel will be closed.
-func (s *snapshotDeserializer) BucketEntries(ctx context.Context, entries chan<- SnapshotEntry) *probe.Error {
```

Comment:
@vadmeste The context provides that functionality.

Reply:
> @vadmeste The context provides that functionality.

The problem with context is that it always provides an error, but I want from BucketEntries to return nil in that case.